packages/server/lib/automation/commands/key_press.ts

```diff
@@ -136,17 +145,27 @@ export async function bidiKeyPress ({ key }: KeyPressParams, client: Client, aut
   }
 
   try {
+    const chars = NamedKeys.includes(inKey) ? [key] : [...key]
+
     await client.inputPerformActions({
       context: autContext,
       actions: [{
         type: 'key',
-        id: `${autContext}-${key}-${idSuffix || Date.now()}`,
-        actions: [
-          { type: 'keyDown', value },
-          { type: 'keyUp', value },
-        ],
+        id: `${autContext}-${inKey}-${idSuffix || Date.now()}`,
```

Comment:
The logical OR encourages the use of `''` as a suffix. Also, will `Date.now()` collide if multiple calls are made in the same millisecond?


```suggestion
        id: `${autContext}-${inKey}-${idSuffix ?? crypto.randomUUID()}`,
```